Example/Info.plist

```diff
@@ -73,5 +73,9 @@
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 	</array>
+	<key>NSCameraUsageDescription</key>
```

Comment:
Could you also update the User Guide with information about why the Host App might need to add these permissions?

Reply:
Done